dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -79,10 +80,72 @@
     private static Persona persona;
     private static Visitor visitor;
 
-    @BeforeClass
-    public static void prepare() throws Exception {
+    @DataProvider
+    public static Object[][] cases() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5025402)